Server/routing/index.js

```diff
@@ -27,39 +27,45 @@ router.get('/', async (ctx) => {
 
 router.get('/t/:miniURL', async (ctx) => {
     // find url in DB or error- does not exist
+    let id = parseInt(ctx.params.miniURL, 36);
+    console.log(id);
+    try {
+        var query = await database.query(
+            'SELECT * FROM public.urls WHERE id = ' + id
+        )
+
+        
+        if (query.rows[0] != null) {
```

Comment:
Instead of if/else branching like this, I prefer to use a bare `if` with an early return for the "failing" case; it makes the logic easier to read. Like:
```
if (query.rows[0] == null) {
  ctx.body = 'Error: Not a valid url';
  return
}

await database.query(
  'UPDATE public.urls SET counter = counter + 1 WHERE id = ' + id
)
ctx.redirect(query.rows[0].original_url);
```

Reply:
Also note that `==` and `===` behave differently. In this case `==` might actually be what you want. But `==` here would also match an empty string, which is probably _not_ what you want.